src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -1106,7 +1108,7 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
                                         $prefix = ''
 
                                         # Don't show line information if script module
-                                        if (($myinv -and $myinv.ScriptName -or $myinv.ScriptLineNumber -gt 1 -or $err.CategoryInfo.Category -eq 'ParserError') -and !($myinv.ScriptName.EndsWith('.psm1', [System.StringComparison]::OrdinalIgnoreCase))) {
+                                        if (($myinv -and $myinv.ScriptName -or $myinv.ScriptLineNumber -gt 1 -or $err.Type -is [System.Management.Automation.ParseException]) -and !($myinv.ScriptName -and $myinv.ScriptName.EndsWith('.psm1', [System.StringComparison]::OrdinalIgnoreCase))) {
```

Comment:
Does `$err` has a `Type` property?

Reply:
It seems that `-is` will simply return `$false` if the member doesn't exist

Reply:
Then how this will cover the original `$err.CategoryInfo.Category -eq 'ParserError'` check?

Also, can you please add comment about when there will be a `Type` member to `$err`, and what type `$err` will be in that case?

Reply:
This was changed quite a bit as when inside the script during formatting, the error instance is NOT a ParseException, but instead an ErrorRecord that contains a ParentContainsErrorRecordException.  So the change here is to check for that since you can't actually get the ParseException while inside this script.